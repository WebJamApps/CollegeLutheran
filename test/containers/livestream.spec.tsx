// import { LiveStream } from 'src/containers/LiveStream/index';

describe('LiveStream', () => {
  it('is true', () => {
    expect(true).toBe(true);
  });
  // it('handles widescreen width', () => {
  //   const result = LiveStream({ width: 932, height: 400 });
  //   expect(result.props.children[2]).toBe(null);
  // });
  // it('handles smaller screen width', () => {
  //   const result = LiveStream({ width: 931, height: 400 });
  //   expect(result.props.children[2]).not.toBe(null);
  // });
  // it('handles middle screen width', () => {
  //   const result = LiveStream({ width: 769, height: 400 });
  //   expect(result.props.children[3].props.style.marginTop).toBe('34vh');
  // });
  // it('handles cellphone screen width', () => {
  //   const result = LiveStream({ width: 600, height: 400 });
  //   expect(result.props.children[3].props.style.marginTop).toBe('38vh');
  // });
});
