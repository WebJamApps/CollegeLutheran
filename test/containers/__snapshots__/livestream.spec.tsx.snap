// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiveStream renders smallscreen 1`] = `
<div
  style={
    {
      "margin": "auto",
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <p>
    The video below will only play during an actual livestream event. To view past services, click to visit our
     
    <a
      href="https://www.youtube.com/@collegelutheranchurch2611/streams"
      rel="noreferrer"
      target="_blank"
    >
      YouTube
    </a>
     
    or
     
    <a
      href="https://www.facebook.com/CollegeLutheranChurch/videos/?ref=page_internal"
      rel="noreferrer"
      target="_blank"
    >
      Facebook
    </a>
     
    pages.
  </p>
  <iframe
    allow="autoplay; fullscreen; picture-in-picture"
    allowFullScreen={true}
    frameBorder="0"
    height="360"
    src="https://player.restream.io/?token=261fc02e28664618bdb68072cfeeb786"
    title="Live Stream Small"
    width="100%"
  />
  <div
    style={
      {
        "marginTop": "38vh",
      }
    }
  >
     
  </div>
</div>
`;

exports[`LiveStream renders widescreen 1`] = `
<div
  style={
    {
      "margin": "auto",
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <h5
    style={
      {
        "marginTop": "0.5rem",
      }
    }
  >
    Welcome to Our Livestream Page
  </h5>
  <p>
    The video below will only play during an actual livestream event. To view past services, click to visit our
     
    <a
      href="https://www.youtube.com/@collegelutheranchurch2611/streams"
      rel="noreferrer"
      target="_blank"
    >
      YouTube
    </a>
     
    or
     
    <a
      href="https://www.facebook.com/CollegeLutheranChurch/videos/?ref=page_internal"
      rel="noreferrer"
      target="_blank"
    >
      Facebook
    </a>
     
    pages.
  </p>
  <iframe
    allow="autoplay; fullscreen; picture-in-picture"
    allowFullScreen={true}
    frameBorder="0"
    height="620"
    src="https://player.restream.io/?token=261fc02e28664618bdb68072cfeeb786"
    title="Live Stream Wide"
    width="930"
  />
  <div
    style={
      {
        "marginTop": "0px",
      }
    }
  >
     
  </div>
</div>
`;
